app/presenters/check_answers/work_items_card.rb

```diff
@@ -0,0 +1,18 @@
+module CheckAnswers
+  class WorkItemsCard < Base
+    attr_reader :work_item_form
+
+    KEY = 'work_items'.freeze
+    GROUP = 'about_claim'.freeze
+
+    def initialize(_claim)
+      # @work_item_form = Steps::WorkItemForm.build(claim)
```

Comment:
is this commented out line intentional ?

Reply:
removed